src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import objects.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
```

Comment:
Perhaps putting the Duke, Parser, Storage, TaskList and UI into a Duke package would make things seem more organised? Maybe adding a header comment to describe what a class does would be better as well :)